Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -114,7 +114,9 @@ class BackForwardListViewController: UIViewController, UITableViewDataSource, UI
         self.currentItem = bfList.currentItem
 
         //error url's are OK as they are used to populate history on session restore.
-        listData = items.filter({return !($0.url.isLocal && ($0.url.originalURLFromErrorURL?.isLocal ?? true))})
+        
+        // disable filter local to manage Mozilla Firefox Start Page
+        listData = items ;  // .filter({return !($0.url.isLocal && ($0.url.originalURLFromErrorURL?.isLocal ?? true))})
```

Comment:
The reason we filter out all locally-hosted pages is because we also use the local web server to display error, reader mode pages, and session restore information. In these cases we want to filter out locally server content. Maybe we can let only the about:home page pass through the filter and let the others still be blocked?

Also, no need for the additional semicolon 😄